Mathlib/Topology/ContinuousFunction/Bounded.lean

```diff
@@ -1516,58 +1508,52 @@ section NormedLatticeOrderedGroup
-    inf_le_left := fun f g => ContinuousMap.le_def.mpr fun _ => inf_le_left
-    inf_le_right := fun f g => ContinuousMap.le_def.mpr fun _ => inf_le_right
-    le_inf := fun f g₁ g₂ w₁ w₂ =>
-      ContinuousMap.le_def.mpr fun _ =>
-        le_inf (ContinuousMap.le_def.mp w₁ _) (ContinuousMap.le_def.mp w₂ _) }
-
-instance semilatticeSup : SemilatticeSup (α →ᵇ β) :=
-  { BoundedContinuousFunction.partialOrder with
-    sup := fun f g =>
-      { toFun := fun t => f t ⊔ g t
-        continuous_toFun := f.continuous.sup g.continuous
-        map_bounded' := by
-          obtain ⟨C₁, hf⟩ := f.bounded
-          obtain ⟨C₂, hg⟩ := g.bounded
-          refine' ⟨C₁ + C₂, fun x y => _⟩
-          simp_rw [NormedAddCommGroup.dist_eq] at hf hg ⊢
-          exact (norm_sup_sub_sup_le_add_norm _ _ _ _).trans (add_le_add (hf _ _) (hg _ _)) }
-    le_sup_left := fun f g => ContinuousMap.le_def.mpr fun _ => le_sup_left
-    le_sup_right := fun f g => ContinuousMap.le_def.mpr fun _ => le_sup_right
-    sup_le := fun f g₁ g₂ w₁ w₂ =>
-      ContinuousMap.le_def.mpr fun _ =>
-        sup_le (ContinuousMap.le_def.mp w₁ _) (ContinuousMap.le_def.mp w₂ _) }
-
-instance lattice : Lattice (α →ᵇ β) :=
-  { BoundedContinuousFunction.semilatticeSup, BoundedContinuousFunction.semilatticeInf with }
-
-@[simp]
-theorem coeFn_sup (f g : α →ᵇ β) : ⇑(f ⊔ g) = ⇑f ⊔ ⇑g := rfl
-#align bounded_continuous_function.coe_fn_sup BoundedContinuousFunction.coeFn_sup
-
-@[simp]
-theorem coeFn_abs (f : α →ᵇ β) : ⇑|f| = |⇑f| := rfl
-#align bounded_continuous_function.coe_fn_abs BoundedContinuousFunction.coeFn_abs
-
-instance : NormedLatticeAddCommGroup (α →ᵇ β) :=
-  { BoundedContinuousFunction.lattice,
-    BoundedContinuousFunction.seminormedAddCommGroup with
+instance instSup : Sup (α →ᵇ β) where
+  sup f g :=
+    { toFun := f ⊔ g
+      continuous_toFun := f.continuous.sup g.continuous
+      map_bounded' := by
+        obtain ⟨C₁, hf⟩ := f.bounded
+        obtain ⟨C₂, hg⟩ := g.bounded
+        refine ⟨C₁ + C₂, fun x y ↦ ?_⟩
+        simp_rw [NormedAddCommGroup.dist_eq] at hf hg ⊢
+        exact (norm_sup_sub_sup_le_add_norm _ _ _ _).trans (add_le_add (hf _ _) (hg _ _)) }
+
+instance instInf : Inf (α →ᵇ β) where
+  inf f g :=
+    { toFun := f ⊓ g
+      continuous_toFun := f.continuous.inf g.continuous
+      map_bounded' := by
+        obtain ⟨C₁, hf⟩ := f.bounded
+        obtain ⟨C₂, hg⟩ := g.bounded
+        refine ⟨C₁ + C₂, fun x y ↦ ?_⟩
+        simp_rw [NormedAddCommGroup.dist_eq] at hf hg ⊢
+        exact (norm_inf_sub_inf_le_add_norm _ _ _ _).trans (add_le_add (hf _ _) (hg _ _)) }
+
+@[simp, norm_cast] lemma coe_sup (f g : α →ᵇ β) : ⇑(f ⊔ g) = ⇑f ⊔ ⇑g := rfl
+#align bounded_continuous_function.coe_fn_sup BoundedContinuousFunction.coe_sup
+
+@[simp, norm_cast] lemma coe_inf (f g : α →ᵇ β) : ⇑(f ⊓ g) = ⇑f ⊓ ⇑g := rfl
+
+instance instSemilatticeSup : SemilatticeSup (α →ᵇ β) :=
+  DFunLike.coe_injective.semilatticeSup _ coe_sup
```

Comment:
I don't know if these changes might have a performance impact. Maybe we could land the name changes and do the other changes separately.

Reply:
This is very easy to answer